core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -5280,7 +5280,25 @@ private static void computeVersion(ServletContext context) {
      * Allows non-privilege escalating configuration permission for a Jenkins instance.  Actions which could result
      * in a privilege  escalation (such as RUN_SCRIPTS) require explicit ADMINISTER permission
      */
-    public static final Permission MANAGE = new Permission(PERMISSIONS, "Manage", Messages._Hudson_ConfigureJenkins_Description(),ADMINISTER, PermissionScope.JENKINS);
+    public static final Permission MANAGE = getManageOrAdministerPermission();/*new Permission(PERMISSIONS,
+                                                             "Manage", Messages._Hudson_ConfigureJenkins_Description
+                                                             (),ADMINISTER, PermissionScope.JENKINS);*/
+
+    /**
+     * As an experimental feature, making the manage permission able to be disabled by default (keep as ADMINISTER), can
+     * be enabled with "jenkins.permission.manage.enabled" system property.
+     * @return ADMINISTER when disabled (default), new permission MANAGE when enabled.
+     */
+    private static Permission getManageOrAdministerPermission() {
+        if(SystemProperties.getBoolean("jenkins.permission.manage.enabled",false)) {
```

Comment:
I think that the modern version of `SystemProperties.getBoolean` is `Boolean.getBoolean`. It seems to be no difference, except that the last one is in the jdk since the 1.8, and the other is a custom method of hudson.

Reply:
ok thank you :)